### PR TITLE
Make mypy green! (added per-line ignores for current errors)

### PR DESCRIPTION
**Pull request by asnytin**
added ignores to prevent mypy check in sqlalchemy packages
added `ignore_missing_imports` flag for sqlalchemy packages and packages without mypy stubs
removed `aiochclient` dependency

### CODE REVIEW

File: lib/dl_connector_chyt/dl_connector_chyt/core/adapters.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     Type,
 )
 
-from aiochclient.http_clients import aiohttp
+import aiohttp
```
Comment (KonstantAnxiety) [inline lib/dl_connector_chyt/dl_connector_chyt/core/adapters.py:11]: Was there a particular reason to use it here in the first place? Or it was a bad import all along

**Comment by KonstantAnxiety** [comment]
@asnytin
Looks awesome) Perhaps you could share a script that you used?

File: lib/dl_formula/pyproject.toml [hunk 2]
```diff
@@ -29,40 +29,20 @@ python-dateutil = ">=2.8.2"
...
+    "graphviz.*",
+    "pytest.*",
+    "pytz.tzinfo.*",
+    "antlr4.*"
+]
 ignore_missing_imports = true
 
 [[tool.mypy.overrides]]
-module = "dl_formula.parser.antlr.gen.DataLensLexer,dl_formula.parser.antlr.gen.DataLensParser,dl_formula.parser.antlr.gen.DataLensVisitor"
+module = "dl_formula.parser.antlr.*"
```
Comment (KonstantAnxiety) [inline lib/dl_formula/pyproject.toml:43]: `"dl_formula.parser.antlr.gen.*"`?

File: lib/dl_api_client/pyproject.toml [hunk 3]
```diff
@@ -42,6 +42,10 @@ disallow_untyped_defs = true
 check_untyped_defs = true
 strict_optional = true
 
+[[tool.mypy.overrides]]
+module = "marshmallow_oneofschema.*"
+ignore_missing_imports = true
+
```
Comment (KonstantAnxiety) [inline lib/dl_api_client/pyproject.toml:48]: Didn't you mention that the new version has become better friends with mypy?

**Review by altvod — CHANGES_REQUESTED** [review]
I would suggest two things:
1. Adding `FIXME` - that way the IDE highlights them (also sometimes you just grep FIXMEs to see what you can fix)
2. Also using some special marker to make all of these comments grep-able

Not sure about no.2, but no.1, is a must

File: lib/dl_api_client/pyproject.toml (hunk 3 above)
Comment (asnytin) [reply lib/dl_api_client/pyproject.toml:48]: You are right. I've forgotten to remove this after rebase.

File: lib/dl_connector_chyt/dl_connector_chyt/core/adapters.py (hunk 1 above)
Comment (asnytin) [reply lib/dl_connector_chyt/dl_connector_chyt/core/adapters.py:11]: I think it was just a mistake (or some kind of PyCharm's incorrect auto-replacement).
There was usual aiohttp module inside.

**Comment by asnytin** [comment]
> I would suggest two things:
> 
>     1. Adding `FIXME` - that way the IDE highlights them (also sometimes you just grep FIXMEs to see what you can fix)
> 
>     2. Also using some special marker to make all of these comments grep-able
> 
> 
> Not sure about no.2, but no.1, is a must

There is already `TODO` mark. We usually is it instead of FIXME

**Review by KonstantAnxiety — APPROVED** [review]
All good
Test fails are not related to the changes in this PR

**Review by altvod — APPROVED** [review]

**Pull request merged**